charts/uptime-kuma/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "1.23.13"
+appVersion: "1.23.14"
```

Comment:
`appVersion` reflects the version of the UptimeKuma Image used. So please keep it at `1.23.13`.

Reply:
I applied your comment. thank you.